src/lib/lldpctl.h

```diff
@@ -738,6 +738,9 @@ typedef enum {
 	lldpctl_k_port_status,	   /**< `(IS,WO)` Operational status of this (local) port */
 	lldpctl_k_port_chassis,	   /**< `(A)` Chassis associated to the port */
 	lldpctl_k_port_ttl,        /**< `(I)` TTL for port, 0 if info is attached to chassis */
+	lldpctl_k_port_vlan_tx_id, /**< `(I)` VLAN ID for TX on port, -1 VLAN disabled */
+	lldpctl_k_port_vlan_tx_prio, /**< `(I)` Priority of VLAN for TX on port, default 0 */
+	lldpctl_k_port_vlan_tx_dei, /**< `(I)` Drop eligible indicator (DEI) of VLAN for TX on port, default 0 */
```

Comment:
They can also be written, so they should be marked as `(I,W)`.

Reply:
good point, I forgot about this.

Reply:
I left only one `lldpctl_k_port_vlan_tx`. Which has `(I,W)` now.
Done.